src/main/scala/chisel3/internal/firrtl/Emitter.scala

```diff
@@ -102,7 +102,7 @@ private class Emitter(circuit: Circuit) {
   private def unindent() { require(indentLevel > 0); indentLevel -= 1 }
   private def withIndent(f: => Unit) { indent(); f; unindent() }
 
-  private val res = new StringBuilder(s"circuit ${circuit.name} : ")
+  private val res = new StringBuilder(s";${Driver.chiselVersionString}\ncircuit ${circuit.name} : ")
```

Comment:
Consider putting the circuit part in a new line, so it doesn't look like it's part of the comment?
```
private val res = new StringBuilder(s";${Driver.chiselVersionString}\n")
res ++= "circuit ${circuit.name} : "
```

Some documentation, like http://www.scala-lang.org/docu/files/collections-api/collections_26.html also construct an empty StringBuilder instead and add everything afterwards, so this might be another option:
```
private val res = new StringBuilder
res ++= s";${Driver.chiselVersionString}\n"
res ++= "circuit ${circuit.name} : "
```
I like the latter, especially in terms of parallel structure.